config/routes.rb

```diff
@@ -26,6 +26,7 @@ def draw_routes(prefix)
         scope 'v2', module: 'v2', as: 'v2' do
           resources :security_guides, only: [:index, :show] do
             resources :profiles, only: [:index, :show], parents: [V2::SecurityGuide]
+            resources :value_definitions, only: [:index], parents: [V2::SecurityGuide]
```

Comment:
@LightOfHeaven1994 do we need to `show` the value definitions over the API? From the QE perspective?